netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,114 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
-| Async Search a| content a| Is now a *primary content* parameter
-| Get Async Result a| jobId a| No longer defaults to `payload`
-| Async Add List a| recordAttributes a| Is now a *primary content* parameter
-| Async Update List a| recordAttributes a| Is now a *primary content* parameter
-| Async Upsert List a| recordAttributes a| Is now a *primary content* parameter
-| Update Invitee Status List a| inviteeStatuses a| Is now a *primary content* parameter
-| Get Posting Transaction Summary a| fields a| Is now a *primary content* parameter
-| Get Posting Transaction Summary a| filters a| Is now a *content* parameter
-| Add File a| attributes a| Is now a *content* parameter
-| Add Record a| attributes a| Is now a *content* parameter.
-| Get Custom Record a| internalId a| No longer defaults to `payload` 
-| Get Deleted Records a| deletedDate a| Is now a *content* parameter
-| Update Record a| attributes a| Is now a *content* parameter
-| Upsert Record a| attributes a| Is now a *content* parameter
+|Operation | Parameter
+
+| Change Email a| newEmail
+| Change Password a| newPasword
+| Get Data Center URLs a| accountId
+| Get Async Result a| jobId
+| Get Custom Record a| internalId 
+|===
+
+The following operations are now *primary content* and *content* parameters:
+
+[%header%autowidth.spread]
+|===
+|Operation | Parameter 
+| Async Search a| content 
+| Async Add List a| recordAttributes 
+| Async Update List a| recordAttributes 
+| Async Upsert List a| recordAttributes
+| Update Invitee Status List a| inviteeStatuses 
+| Get Posting Transaction Summary 
+| Get Posting Transaction Summary 
+| Add File a| attributes 
+| Add Record a| attributes  
+| Get Deleted Records a| deletedDate 
+| Update Record a| attributes 
+| Upsert Record a| attributes 
 |===
 
 [[changed_sources]]
 == Changed Sources
 
-The three message sources were renamed.
+The three message sources were renamed:
 
 [%header%autowidth.spread]
 |===
 |Source name |Old XML name | New XML name
 
-| On New Object | `<netsuite:new-object-trigger ...` | `<netsuite:new-object-listener ...`
-| On Modified Object |  `<netsuite:modified-trigger ...` | `<netsuite:modified-listener ...`
-| On Deleted Object | `<netsuite:deleted-trigger ...` | `<netsuite:deleted-listener ...`
+| On New Object | `<netsuite:new-object-trigger` | `<netsuite:new-object-listener`
+| On Modified Object |  `<netsuite:modified-trigger` | `<netsuite:modified-listener`
+| On Deleted Object | `<netsuite:deleted-trigger` | `<netsuite:deleted-listener`
 |===
 
 == Upgrade Steps
 
-If you are upgrading from an earlier version of the connector, a small popup appears in the bottom right corner of Anypoint Studio with the `Updates Available` message.
+If you are upgrading from an earlier version of the connector:
 
-Click the pop-up and follow the prompts to install the latest version.
+. A small pop-up message appears in the bottom right corner of Anypoint Studio with the `Updates Available` message.
```

Comment:
This is not a step, because there is no user action in it.
Which leaves just one step...which is okay, but we can also work around it...hmmm.... Something like...:

Upgrade from an Earlier Version of the Connector

"You can now click a new `Updates Available` pop-up message in the bottom right corner of Anypoint Studio to upgrade your connector.

You can also continue to follow these steps..."